docs/merge-sections.html

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: Merge Sections
+---
+
+<div id="content-wrap">
+ <h1>Merge Sections Documentation</h1>
+ <p>Follow documentation for this Features</p>
+</div>
```

Comment:
I think this is still 1 space indentation. Please change to 2 space indentations.

Reply:
All fixed!